magento/adobe-stock-integration
# Added Content Status and Store View filters

### Description (*)
Added Content Status and Store View filters.

1. Depends on https://github.com/magento/magento2/pull/29058

### Related Pull Requests
https://github.com/magento/magento2/pull/29058


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1463 
2. Fixes magento/adobe-stock-integration#1464

### Manual testing scenarios (*)

1. Assign an asset to a content (product, category, page or block)
2. Go to `Content` -> `Media Gallery` and filter by `Content Status` or `Store View`
3. The filter output should be assets that are used on content and the content matches the filter condition.
